composer.json

```diff
@@ -85,7 +86,130 @@
+        "url": "https://github.com/bahamas10/css-color-names/archive/v1.0.1.zip"
+      },
+      "d3": {
+        "url": "https://github.com/mbostock-bower/d3-bower/archive/v3.4.11.zip",
+        "ignore": [".DS_Store", ".git", ".gitignore", ".npmignore", ".travis.yml", "Makefile", "bin", "component.json", "index.js", "lib", "node_modules", "package.json", "src", "test"]
+      },
+      "d3-3.5.x": {
+        "url": "https://github.com/mbostock-bower/d3-bower/archive/v3.5.17.zip"
+      },
+      "datatables": {
+        "url": "https://github.com/DataTables/DataTables/archive/1.10.19.zip",
+        "ignore": ["/.*", "examples", "media/unit_testing", "composer.json", "dataTables.jquery.json", "package.json"]
+      },
+      "dc-2.1.x": {
+        "url": "https://github.com/NickQiZhu/dc.js/archive/2.1.10.zip",
+        "ignore": [".*", "style", "web", "*.json", "regression", "scripts", "spec", "src", "docs", "grunt", "Gruntfile.js", "Changelog.md", "welcome.md", "class-hierarchy.dot", "index.js", "CONTRIBUTING.md", "LICENSE_BANNER", "AUTHORS"]
+      },
+      "es6-promise": {
+        "url": "https://github.com/components/es6-promise/archive/v4.2.4.zip"
+      },
+      "font-awesome": {
+        "url": "https://github.com/FortAwesome/Font-Awesome/archive/v4.7.0.zip",
+        "ignore": ["*/.*", "*.json", "src", "*.yml", "Gemfile", "Gemfile.lock", "*.md"]
+      },
+      "google-code-prettify": {
+        "url": "https://github.com/tcollard/google-code-prettify/archive/v1.0.5.zip",
+        "ignore": ["closure-compiler", "js-modules", "tests", "yui-compressor", "Makefile"]
+      },
+      "jquery": {
+        "url": "https://github.com/civicrm/jquery/archive/1.12.4-civicrm-1.2.zip"
+      },
+      "jquery-ui": {
+        "url": "https://github.com/components/jqueryui/archive/1.12.1.zip"
+      },
+      "jquery-validation": {
+        "url": "https://github.com/jquery-validation/jquery-validation/archive/1.13.1.zip",
+        "ignore": [".*", "node_modules", "bower_components", "test", "demo", "lib"]
+      },
+      "jstree": {
+        "url": "https://github.com/vakata/jstree/archive/3.3.8.zip",
+        "ignore": [".*", "docs", "demo", "libs", "node_modules", "test", "libs", "jstree.jquery.json", "gruntfile.js", "package.json", "bower.json", "component.json", "LICENCE-MIT", "README.md"]
+      },
+      "lodash-compat": {
+        "url": "https://github.com/lodash/lodash-compat/archive/3.0.1.zip",
+        "ignore": [".*", "*.log", "*.md", "component.json", "package.json", "node_modules"]
+      },
+      "phantomjs-polyfill": {
+        "url": "https://github.com/conversocial/phantomjs-polyfill/archive/v0.0.2.zip"
+      },
+      "qunit": {
+        "url": "https://github.com/jquery/qunit/archive/v1.10.0.zip"
+      },
+      "select2": {
+        "comment": "This is pegged to a branch, which may have caching+update issues. Should switch to a tag.",
+        "url": "https://github.com/colemanw/select2/archive/stable/3.5.zip"
```

Comment:
@colemanw Would you mind making a tag in the `colemanw/select2` repo? Maybe something like

```
git tag v3.5-civicrm-1.0 90d4b9ee388b33e17c9f58a2866a4765f00b9a55
```